Assets/Scripts/EditorTests/HealthTests.cs

```diff
@@ -0,0 +1,118 @@
+using NUnit.Framework;
+using SS3D.Systems.Health;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+
+namespace EditorTests.Health
+{
+    public class HealthTests
+    {
+        #region Class variables
+        /// <summary>
+        /// Toggles whether any test-specific Debug.Log's are displayed in the console.
+        /// </summary>
+        private bool SHOW_DEBUG = false;
+        #endregion
+
+        #region Test set up
+        [SetUp]
+        public void SetUp()
+        {
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+        }
+        #endregion
+
+        #region Tests
+
+        /// <summary>
+        /// Test to confirm that interactions can only be commenced when stamina is greater than zero, and will otherwise fail.
+        /// </summary>
+        [Test]
+        [TestCase(9.99f, true)]
+        [TestCase(10.01f, false)]
+        public void CanCommenceInteractionOnlyWhenStaminaIsGreaterThatZero(float staminaToDeplete, bool expectedResult)
+        {
+
+            // ARRANGE
```

Comment:
It's great that you're writing tests!

I think we can leave out the "step" comments, what do you think?

Reply:
They're helpful for me as I develop them, but yeah, it doesn't need to be in the production code for tests this simple. I will remove them.